### PR TITLE
[15.0][MIG] hr_timesheet_report_not_valued

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

File: hr_timesheet_report_not_valued/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Timesheet Report Not Valued",
+    "summary": """
+        Adds a report that hides the time field.
+    """,
+    "version": "12.0.1.0.0",
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/__manifest__.py]: Wrong version, please fix it. You should also re-generate README info

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment, and also you should noticed this when installing the addon:

`warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)`

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0 and try to apply it:

imagen

File: hr_timesheet_report_not_valued/__manifest__.py (hunk 1 above)
Comment (adriresu) [reply hr_timesheet_report_not_valued/__manifest__.py]: Changed and updated manifest version

Comment (adriresu) [reply hr_timesheet_report_not_valued/__manifest__.py]: Changed and updated <report> tag. Also updated README

File: hr_timesheet_report_not_valued/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4.3\n"
+"Last-Translator: \n"
+"Language: es_ES\n"
+
+#. module: hr_timesheet_report_not_valued
+#: model:ir.actions.report,name:hr_timesheet_report_not_valued.timesheet_not_valued_report
+msgid "Timesheet Entries Not Valued"
+msgstr "Detalle de los partes de horas no valorado"
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/i18n/es.po]: v15 translations have changed at this point, and we have this inconsistency:

imagen

So, a better translation, for coherence, should be the following:
```suggestion
msgstr "Entradas de los partes de horas no valoradas"
```

File: hr_timesheet_report_not_valued/report/hr_timesheet_report.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <t t-call="hr_timesheet.report_timesheet"/>
+    </template>
+
+    <record id="timesheet_not_valued_report" model="ir.actions.report">
+        <field name="name">Timesheet Entries Not Valued</field>
+        <field name="model">account.analytic.line</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">hr_timesheet_report_not_valued.report_timesheet_not_valued</field>
+        <field name="report_file">report_timesheet</field>
+        <field name="binding_model_id" ref="account.analytic.line" />
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/report/hr_timesheet_report.xml]: This is wrong and fires an error when installing (`ref` attribute in XML data shoud point to a valid XML identifier). Try this:
```suggestion
        <field
            name="binding_model_id"
            ref="hr_timesheet.model_account_analytic_line"
        />
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments, squash them into migration commit

**Comment by adriresu** [comment]
Changes done and squashed. :)

**Review by dalonsod — APPROVED** [review]
:+1:

**Pull request merged**